src/main/java/frc/robot/RobotContainer.java

```diff
@@ -7,9 +7,12 @@
 import edu.wpi.first.wpilibj.GenericHID;
 import edu.wpi.first.wpilibj.XboxController;
 import frc.robot.commands.SwerveTeleopCommand;
+//import frc.robot.subsystems.AssistedBalance;
```

Comment:
Go ahead and remove this, if we aren't using it.

Reply:
Done